### PR TITLE
enhancement/category

### PR DESCRIPTION
**Pull request by allenazamoras**
- [x] changed `category` from attribute to models
- [x] pushed `category` migration files
- [x] fixed `list_by_category` endpoint

### CODE REVIEW

File: livestream/views.py [hunk 1]
```diff
@@ -56,9 +60,9 @@ def create(self, request, *args, **kwargs):
                              session_id=session.session_id,
                              owner=request.user,
                              helper=None,
-                             detail=req['detail'],
-                             category=req['category'])
+                             detail=req['detail'])
         new_session.save()
+        new_session.category.add(req['category'])
```
Comment (aldnav) [inline livestream/views.py]: Is `req['category']` a category instance?

Comment (allenazamoras) [reply livestream/views.py]: category id sir

**Comment by aldnav** [comment]
It doesn't raise a TypeError? Is it not that. add expects a Category
instance? Also, what happens when you pass a non-existent id/instance? Im
on mobile so please confirm. 😊

**Comment by allenazamoras** [comment]
maka accept raman siya og id sir, although i'd probably change it para maka check if ni exist bajud ang category

**Comment by aldnav** [comment]
@allenazamoras :+1:

**Pull request merged**